asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py

```diff
@@ -231,7 +237,13 @@ def write_traj_visualization(self, traj, system, path):
         p.cmd.hide("sticks", "(elem C extend 1) and (elem H)")
         p.cmd.color("pink", "elem C and sele")
 
-        p.cmd.set_view(self.view_coords)
+        for subpocket_name, color in self.color_dict.items():
```

Comment:
This has reduced performance on GIF generation but shouldn't take more than  20 mins on a Lilac node.